bin/run-command

```diff
@@ -118,104 +118,3 @@ printf " ... %s\n" "${aws_args[@]}"
-echo "  task_id=${ecs_task_id}"
-echo "  log_stream=${log_stream}"
-
-num_retries_waiting_for_logs=0
-while true; do
-  num_retries_waiting_for_logs=$((num_retries_waiting_for_logs + 1))
-  if [ "${num_retries_waiting_for_logs}" -eq 20 ]; then
-    echo "Timing out task ${ecs_task_id} waiting for logs"
-    exit 1
-  fi
-  is_log_stream_created=$(aws logs describe-log-streams --no-cli-pager --log-group-name "$log_group" --log-stream-name-prefix "$log_stream" | jq -r '.logStreams | length')
-  if [ "$is_log_stream_created" == "1" ]; then
-    break
-  fi
-  sleep 5
-  echo -n "."
-done
-echo
-echo
-
-# Tail logs until task stops using a loop that polls for new logs.
-# The reason why we don't use `aws logs tail` is because that command is meant
-# for interactive use. In particular, it will wait forever for new logs, even
-# after a task stops, until the user hits Ctrl+C. And the reason why we don't
-# wait until the task completes first before fetching logs is so that we can
-# show logs in near real-time, which can be useful for long running tasks.
-echo "::group::Tailing logs until task stops"
-echo "  log_group=${log_group}"
-echo "  log_stream=${log_stream}"
-echo "  task_start_time_millis=${task_start_time_millis}"
-# Initialize the logs start time filter to the time we started the task
-logs_start_time_millis="${task_start_time_millis}"
-while true; do
-  # Print logs with human readable timestamps by fetching the log events as JSON
-  # then transforming them afterwards using jq
-  log_events=$(aws logs get-log-events \
-    --no-cli-pager \
-    --log-group-name "${log_group}" \
-    --log-stream-name "${log_stream}" \
-    --start-time "${logs_start_time_millis}" \
-    --start-from-head \
-    --no-paginate \
-    --output json)
-  # Divide timestamp by 1000 since AWS timestamps are in milliseconds
-  echo "${log_events}" | jq -r '.events[] | ((.timestamp / 1000 | strftime("%Y-%m-%d %H:%M:%S")) + "\t" + .message)'
-
-  # If the task stopped, then stop tailing logs
-  last_task_status=$(aws ecs describe-tasks --cluster "${cluster_name}" --tasks "${task_arn}" --query "tasks[0].containers[?name=='${container_name}'].lastStatus" --output text)
-  if [ "${last_task_status}" = "STOPPED" ]; then
-    break
-  fi
-
-  # If there were new logs printed, then update the logs start time filter
-  # to be the last log's timestamp + 1
-  last_log_timestamp=$(echo "${log_events}" | jq -r '.events[-1].timestamp')
-  if [ "${last_log_timestamp}" != "null" ]; then
-    logs_start_time_millis=$((last_log_timestamp + 1))
-  fi
-
-  # Give the application a moment to generate more logs before fetching again
-  sleep 1
-done
-echo "::endgroup::"
-echo
-
-container_exit_code=$(
-  aws ecs describe-tasks \
-    --cluster "${cluster_name}" \
-    --tasks "${task_arn}" \
-    --query "tasks[0].containers[?name=='${container_name}'].exitCode" \
-    --output text
-)
-
-if [[ "${container_exit_code}" == "null" || "${container_exit_code}" != "0" ]]; then
```

Comment:
Wouldn't removing all of this mean we no longer error if migrations fail to run? How will be verify migrations ran successfully every run?

Reply:
@chouinar this is still in draft, I added that back :)

Reply:
kay -now- its ready